aptos-move/aptos-aggregator/src/types.rs

```diff
@@ -26,6 +26,20 @@ impl AggregatorID {
     }
 }
 
+// Used for ID generation and for Value exchange.
+impl From<u64> for AggregatorID {
+    fn from(value: u64) -> Self {
+        Self::new(value)
+    }
+}
+
+// Used for Value exchange.
+impl From<AggregatorID> for u64 {
+    fn from(id: AggregatorID) -> Self {
```

Comment:
Should be returning u64.

Reply:
It compiled though, Rust magic I guess 😂

Reply:
well Self is u64 - because this is implementation of function for u64 type :)